src/constants/dropdown.ts

```diff
@@ -0,0 +1,12 @@
+export type PriceSortOption = '가격이 낮은 순' | '가격이 높은 순';
+export type ReserveSortOption = '전체' | '예약 신청' | '예약 취소' | '예약 승인' | '예약 거절' | '체험 완료';
+
+export const PRICE_LIST: PriceSortOption[] = ['가격이 낮은 순', '가격이 높은 순'];
+export const RESERVE_LIST: ReserveSortOption[] = [
+  '전체',
+  '예약 신청',
+  '예약 취소',
+  '예약 승인',
+  '예약 거절',
+  '체험 완료',
+];
```

Comment:
@qooktree1 참고로 똑같은 글자를 다시 쓰기 싫다면 이런 방식도 있습니다. 
```suggestion
export type PriceSortOption = (typeof PRICE_LIST)[number];
export type ReserveSortOption = (typeof RESERVE_LIST)[number];

export const PRICE_LIST = ['가격이 낮은 순', '가격이 높은 순'] as const;
export const RESERVE_LIST = ['전체', '예약 신청', '예약 취소', '예약 승인', '예약 거절', '체험 완료'] as const;

```
이렇게 써주면 원하시는대로 타입 추론이 잘 됩니다.
<img width="556" alt="스크린샷 2024-01-28 오후 7 38 17" src="https://github.com/TripTripNow/TodayTrip/assets/112458620/ce5fd21c-053f-41af-aa2d-701db6fdce0e">